pkg/azure/aks/machine_store.go

```diff
@@ -56,9 +56,9 @@ type VirtualMachineInfo struct {
 	OperatingSystem MachineOperatingSystem
 	Priority        MachinePriority
 
-	NumOfCores     int32
-	MemoryInMiB    int32
-	OsDiskSizeInMB int32
+	NumOfCores     float64
+	MemoryInMiB    float64 // Note, the Azure Docs say MiB, the golang docs say MB, we're going with the Azure Docs :nervous:
```

Comment:
It's pretty easy to spot this, when we introduce it to the cost metrics ;-) ... we will find out